BeeSwift/AppDelegate.swift

```diff
@@ -140,19 +137,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
     // MARK: - UNUserNotificationCenterDelegate
     
     func userNotificationCenter(_ center: UNUserNotificationCenter, willPresent notification: UNNotification, withCompletionHandler completionHandler: @escaping (UNNotificationPresentationOptions) -> Void) {
-        switch notification.request.identifier {
-        case JSONGoal.unlockNotificationIdentifier:
-            // about to present a notification, a reminder to unlock the device
-            // yet the app is active, thus we can abandon the notification
-            completionHandler([])
-        default:
-            completionHandler([.alert, .sound, .badge])
-        }
+        completionHandler([.alert, .sound, .badge])
```

Comment:
This whole section was introduced in Commit 1b33ac579d3e59bdd and probably could be removed along with the rest of the stuff removed in this PR.

As is now, the presentation options for notifications shown while the app is in the foreground are overwritten. We probably could just use their defaults, which can be achieved by not implementing this method `func userNotificationCenter ... willPresent ...`.

https://developer.apple.com/documentation/usernotifications/unusernotificationcenterdelegate/1649518-usernotificationcenter